apps/src/blockly/addons/cdoFieldFlyout.js

```diff
@@ -0,0 +1,135 @@
+import GoogleBlockly from 'blockly/core';
+import CdoBlockFlyout from './cdoBlockFlyout';
+
+export default class CdoFieldFlyout extends GoogleBlockly.Field {
+  /**
```

Comment:
Could you add a comment similar to the one you included for `CdoBlockFlyout` that explains the relationship between the two classes? In the PR description, it was helpful to read that 'the custom field CdoFieldFlyout can be thought of as a container for a flyout UI (CdoBlockFlyout)'.